crate/operator/handlers/handle_create_grand_central.py

```diff
@@ -52,3 +56,11 @@ async def create_grand_central(
             ),
             id="create_grand_central",
         )
+
+        kopf.register(
+            fn=subhandler_partial(
+                create_grand_central_user, namespace, name, cratedb["metadata"], logger
+            ),
+            id="create_grand_central_user",
+            backoff=config.BOOTSTRAP_RETRY_DELAY,
```

Comment:
I suspect you might want to add a `config.BOOTSTRAP_RETRY_DELAY` to the above subhandler too, because it will always fail initially, because of the missing secret.